lib/cosi/cosi.go

```diff
@@ -191,15 +191,25 @@ func (c *Cosi) CreateResponse() (*Response, error) {
 
 // Response generates the response from the commitment, challenge and the
 // responses of its children.
-func (c *Cosi) Response(responses []*Response) (*Response, error) {
-	// create your own response
-	if err := c.genResponse(); err != nil {
-		return nil, err
+// if respond == true, will also add our response
+func (c *Cosi) Response(respond bool, responses []*Response) (*Response, error) {
```

Comment:
Also, just to be sure, this behavior is not to be implemented now in the crypto library right ? Because this behaviors means that a node can refuse to sign after a commit, that means it has to pass in the final signature its public key and its commit. 
In the current design of dedis/crypto/cosi, we only have the *mask* which represents the missing public keys to substract, but there's nothing to handle a failure after a commit.